src/components/OptionsSelector.js

```diff
@@ -153,15 +154,24 @@ class OptionsSelector extends Component {
         }
     }
 
-    componentDidUpdate(prevProps) {
+    componentDidUpdate(prevProps, prevState) {
         if (_.isEqual(this.props.sections, prevProps.sections)) {
             return;
         }
 
+        const newOptions = OptionsListUtils.flattenSections(this.props.sections);
         // eslint-disable-next-line react/no-did-update-set-state
         this.setState({
-            allOptions: OptionsListUtils.flattenSections(this.props.sections),
+            allOptions: newOptions,
         });
+
+        if (newOptions.length > 0 && this.focusManager) {
+            const prevFocusedIndex = this.focusManager.getFocusedIndex();
+            const prevFocusedItem = prevState.allOptions[prevFocusedIndex];
+            const newFocusedIndex = Math.max(_.findIndex(newOptions, option => option && option.login === prevFocusedItem.login), 0);
```

Comment:
@roryabraham I'm seeing the error boundary with the new changes.

Steps
1. Open search page `Cmd + K`
2. Search for a garbage value (so that you don't see any results. eg: "fdasf2342421fds")
3. Keep pressing backspace.

In console you'll see that `prevFocusedItem` is `undefined`.
<img width="890" alt="Screen Shot 2022-04-13 at 5 55 15 PM" src="https://user-images.githubusercontent.com/29673073/163181701-3160e52c-f18f-4488-bedb-fd5233ee30f0.png">

Reply:
Great catch, fixed!